script/JSON_validator/presentation_definition_validator.py

```diff
@@ -0,0 +1,94 @@
+import json
```

Comment:
this is not what I have explicitly asked

we need a loader of the json schema validator using pydantic, as the provided example. We don't need static code doing static validation since the json validation schema may change